.vscode/settings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "cSpell.enabled": false,
+  "cSpell.words": ["businessnjgovnavigator"],
+  "editor.codeActionsOnSave": ["source.organizeImports", "source.formatDocument", "source.fixAll.eslint"],
+  "editor.formatOnSave": true,
+  "editor.rulers": [80, 110],
+  "editor.wordWrap": "wordWrapColumn",
+  "editor.wordWrapColumn": 110,
+  "eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
+  "explorer.fileNesting.enabled": true,
+  "explorer.fileNesting.patterns": {
+    "*.ts": "$(capture).test.ts, $(capture).test.tsx",
+    "*.tsx": "$(capture).test.ts, $(capture).test.tsx"
+  },
+  "files.exclude": {
+    "**/node_modules": false,
+    "node_modules": true,
+    "**/.next": true,
+    "*[!test]**/node_modules": true
+  },
+  "files.trimTrailingWhitespace": true,
+  "git.allowForcePush": true,
+  "git.autofetch": true,
+  "git.autoStash": true,
+  "git.enableSmartCommit": true,
+  "git.fetchOnPull": true,
+  "git.mergeEditor": true,
+  "git.terminalGitEditor": true,
```

Comment:
@seidior I'd like to revisit some of these adds. Particularly the changes that modify git workflow and the editor behavior. I see where these could be helpful, but I'm finding some get more in my way than help. I'm fine with offering setting recommendations, but I don't see a reason why the repo should dictate workflow.

I'm feeling differently about this than I did a few days ago after working with the settings in place.

Reply:
Thank you so much for giving this time, @jphechter. I'll remove the offending lines in a follow-up chore; they were a last minute add that I don't feel strongly about.